@types/web-crypto.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "crypto" {
+  namespace webcrypto {
+    const subtle: SubtleCrypto;
```

Comment:
See this is a thing in latest 16
https://unpkg.com/browse/@types/node@16.18.11/crypto.d.ts